<launch>

    <arg name="twist_topic" default="/twist_controller/command"/>
    <arg name="pose_topic" default="/target_frame"/>
    <arg name="wrench_topic" default="/target_wrench"/>
    <arg name="arm_topic" default="/arm_controller/command"/>
    <arg name="gripper_topic" default="/gripper_controller/command"/>
    <arg name="publishing_rate" value="50"/>

    <!-- Numerically integrate twists into a pose -->
    <node name="trossen_joy" pkg="morpheus_teleop" type="trossen_joy.py" output="screen">
        <param name="pose_topic" value="$(arg pose_topic)"/>
        <param name="wrench_topic" value="$(arg wrench_topic)"/>
        <param name="arm_topic" value="$(arg arm_topic)"/>
        <param name="gripper_topic" value="$(arg gripper_topic)"/>
        <param name="publishing_rate" value="$(arg publishing_rate)"/>
    </node>

</launch>